financialaid/models.py

```diff
@@ -138,7 +106,11 @@ class FinancialAid(TimestampedModel):
     tier_program = models.ForeignKey(TierProgram, null=False)
     status = models.CharField(
         null=False,
-        choices=[(status, status) for status in FinancialAidStatus.ALL_STATUSES],
+        choices=[
+            (status, status) for status in FinancialAidStatus.ALL_STATUSES
+            # The rejected status has been temporarily disabled as a valid status for FinancialAid
```

Comment:
Please revert this change unless it's necessary for the work in this PR. We should not be putting in 'temporary' changes, especially right before a release. We should remove `rejected` with a separate PR